### PR TITLE
Fixed SvelteKit related issues

### PR DESCRIPTION
**Pull request by alec-c4**
### Description

Fixed SvelteKit related issues:
- removed pluginSearchDirs from prettier config, because of https://prettier.io/blog/2023/07/05/3.0.0.html#plugin-search-feature-has-been-removed-14759httpsgithubcomprettierprettierpull14759-by-fiskerhttpsgithubcomfisker
- fixed `Cannot find base config file "./.svelte-kit/tsconfig.json"` - https://github.com/sveltejs/kit/issues/5390
- updated deps

### Testing Instructions

```sh
npx create-turbo@latest -e with-svelte
```

### CODE REVIEW

**Comment by alec-c4** [comment]
@anthonyshew @NicholasLYang Hey! May I ask you check what's wrong with PR?

**Comment by tknickman** [comment]
Hey @alec-c4 could you fix the conflicts here? Then we should be able to get it in 👍🏼

**Comment by alec-c4** [comment]
> Conflicts need to be resolved

Hey @tknickman — I've resolved conflicts, but there are some issues with deployment to the Vercel :(  

SCR-20240405-pydt

File: examples/with-svelte/apps/docs/package.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
 	"version": "0.0.1",
 	"private": true,
 	"type": "module",
+  "prepare": "svelte-kit sync",
```
Comment (mcmxcdev) [inline examples/with-svelte/apps/docs/package.json:6]: This is misplaced, needs to go into `scripts`

Comment (alec-c4) [reply examples/with-svelte/apps/docs/package.json]: @mcmxcdev fixed, thanks!

**Review by tknickman — COMMENTED** [review]
It looks like the pnpm lock file for the example is out of date and that's why the install is failing on Vercel. A pnpm install should fix it up!

**Comment by anthonyshew** [comment]
Looks like this one has fallen stale. I believe it's also been superceded by other work.

Feel free to open a PR with any more improvements you see with today's iterations!

**Pull request closed without merging**